test/azure/terraform_azure_example_test.go

```diff
@@ -1,12 +1,11 @@
-// +build azure
+// +build azure azureslim,compute
```

Comment:
What are the new tags?

Reply:
@rguthriemsft bought this up so we can have smaller CI builds under common groups of tests. Another example can be seen with azureslim,network on a few of those network tests.